addons/options/package.json

```diff
@@ -23,7 +23,7 @@
     "prepare": "node ../../scripts/prepare.js"
   },
   "dependencies": {
-    "@storybook/addons": "4.0.8",
+    "@storybook/addons": "4.0.12",
```

Comment:
Why is this an exact dependency?

Reply:
AFAIK we have exact deps to our pkgs everywhere.

Reply:
@igor-dv this is the problem